src/App.tsx

```diff
@@ -12,38 +17,90 @@ export const goodsFromServer = [
   'Fish',
   'Honey',
   'Jam',
-  'Garlic',
 ];
 
 export const App: React.FC = () => {
+  const [sortField, setSortField] = useState<SortField | ''>('');
+  const [reversed, setReversed] = useState<boolean>(false);
+  const elems = [...goodsFromServer];
+
+  if (sortField) {
+    elems.sort((elem1: string, elem2: string) => {
+      switch (sortField) {
+        case SortField.Alpha:
+          return elem1.localeCompare(elem2);
+        case SortField.Length:
+          return elem1.length - elem2.length;
+        default:
+          return 0;
+      }
+    });
+  }
+
+  if (reversed) {
+    elems.reverse();
+  }
+
+  const handleReset = () => {
+    setSortField('');
+    setReversed(false);
+  };
+
   return (
     <div className="section content">
       <div className="buttons">
-        <button type="button" className="button is-info is-light">
+        <button
+          type="button"
+          className={
+            sortField === SortField.Alpha
+              ? 'button is-info'
+              : 'button is-info is-light'
+          }
```

Comment:
it is better to use libraries like `classnames` or `clsx` for conditional className